nap/fields.py

```diff
@@ -108,6 +109,14 @@ def descrub_value(self, val, for_read=False):
         return datetime.date.strftime(val, dt_format)
 
 
+class DecimalField(Field):
+    def scrub_value(self, val):
+        return Decimal(str(val)) if val else None
+
+    def descrub_value(self, val, for_read=False):
+        return unicode(val) if val else None
```

Comment:
Could we safely use `str()` here instead of `unicode()`? I ask because `unicode` no longer exists in Python 3.